app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-feature android:name="android.hardware.camera.front" android:required="false" />
```

Comment:
Consider using 
```xml
<uses-feature android:name="android.hardware.camera.any" android:required="false" />
```
in case the device has more than 2 camera